src/test/java/com/education/service/ClassServiceTest.java

```diff
@@ -4,97 +4,94 @@
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.transaction.annotation.Transactional;
+import org.springframework.test.context.jdbc.Sql;
 
 import java.util.List;
 
 import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.context.jdbc.Sql.ExecutionPhase.AFTER_TEST_METHOD;
 
+@Sql(executionPhase = AFTER_TEST_METHOD, value = "/sql/cleanup.sql")
 @SpringBootTest
-@Transactional
 public class ClassServiceTest {
+
     @Autowired
     private ClassService classService;
 
     @Test
     public void save() {
-        Class room = new Class();
-        room.setClassNumber("125");
-        room.setCapacity(45);
-        classService.save(room);
-        Long roomId = room.getId();
+        Class aClass = new Class();
+        aClass.setClassNumber("125");
+        aClass.setCapacity(45);
+        classService.save(aClass);
 
-        Class roomById = classService.getById(roomId);
+        Class ClassById = classService.getById(aClass.getId());
```

Comment:
Not needed. Just use the return value of classService.save method. It returns an object with id

Reply:
`ClassById` -> `classById`